manifests/dynamic-networks-controller.yaml

```diff
@@ -0,0 +1,122 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: dynamic-networks-controller
+rules:
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources:
+      - '*'
```

Comment:
I would list required rights explicitly from the beginning (doing that later once the project grows may be a headache)

Reply:
Done.